hs_discover/views.py

```diff
@@ -1,57 +1,111 @@
 import json
 
+from django.core.paginator import Paginator
 from django.shortcuts import render
 from django.views.generic import TemplateView
 from haystack.query import SearchQuerySet
+from rest_framework.response import Response
+from rest_framework.views import APIView
 
 
-# TODO error handling, validation, analytics
 class SearchView(TemplateView):
 
     def get(self, request, *args, **kwargs):
-        from django.template.defaultfilters import date, time
-        q = request.GET.get('q') if request.GET.get('q') else ""
+        return render(request, 'hs_discover/index.html', {})
```

Comment:
Considering this is a TemplateView, probably should use template_name and not override get.  Or better, remove this class already and just use TemplateView and specify the template_name in the as_view method.  https://docs.djangoproject.com/en/1.11/topics/class-based-views/#simple-usage-in-your-urlconf